embrace-android-delivery/build.gradle.kts

```diff
@@ -12,5 +12,6 @@ apiValidation.validationDisabled = true
 
 dependencies {
     implementation(project(":embrace-android-payload"))
+    implementation(project(":embrace-android-core"))
     testImplementation(project(":embrace-android-payload"))
```

Comment:
Do we still need the `testImplementation` dep now that it's also s `implementation` dep?

Reply:
Removed this as it's unnecessary